src/index.tsx

```diff
@@ -19,14 +20,8 @@ const ChariotConnect: React.FC<ChariotConnectProps> = ({
     onExit = noop,
     disabled = false,
 }) => {
+    useScript({ src: 'https://cdn.givechariot.com/chariot-connect.umd.js' });
```

Comment:
hmm... useScript provides [loading, error] return values, we should maybe utilize those in interesting ways. @magaldima what do you think? Maybe a followup?

Reply:
I think we should address this concern now. @themaxgoldman I think it's important to respect those values since we wouldn't want the button to just not load if the script fails to load and we also want to render the button only after the script is finished loading

Reply:
Okay I addressed this. Now the button won't load until the script is loaded and loading and error are dependencies of `useEffect`.

Reply:
@magaldima I misunderstood. What behavior would you want if the script fails to load?

Reply:
Refactored so that the `useEffect` function returns early when loading or error and we log on error.